UnityProject/Assets/Scripts/UI/PlayerHealthUI.cs

```diff
@@ -23,6 +23,8 @@ public class PlayerHealthUI : MonoBehaviour
 	void Awake()
 	{
 		DisableAll();
+		///fixes Healthdoll sprite
+		baseBody.transform.Rotate( 0, 180, 0, Space.Self);
```

Comment:
This can probably done in the `PlayerHealth_UI_Hud` prefab instead by flipping the y rotation of `DamageMonitor_Base` by 180 degrees.